src/Atoms/MainBtn/MainBtnStyled.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import COLORS from '../../Styles/varsStyledComponents';
+import FONTS from '../../Styles/fontsStyledComponents';
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const MainBtnStyled = styled.button`
+  width: 343px;
+  height: 50px;
+  color: ${(props) =>
+    props.primary
+      ? COLORS.white
+      : props.secondary
+      ? COLORS.primary601
+      : COLORS.grey800};
+  ${FONTS.button};
+  background: ${(props) =>
+    props.primary
+      ? COLORS.primary501
```

Comment:
That's not what I meant actually. I mean ternary operators are simple YES:NO solution, they should be used as that. If you have to use: YES: NO? YES: NO, its not a way to go. See if you can solve it in another way, withuot nesting the expression.